README.md

```diff
@@ -200,7 +200,63 @@ published on Hackage, or part of Stackage snapshots, using Bazel.
 
 [hazel]: https://github.com/formationai/hazel
 
-## Troubleshooting `rules_haskell` development
+## Troubleshooting
+
+### `as`: no such file or directory
+
+If you see error messages complaining about missing `as` executable:
+
+```
+cc: error trying to exec 'as': execvp: No such file or directory
+`cc' failed in phase `Assembler'. (Exit code: 1)
+```
+
+It means that your `gcc` cannot find `as` by itself. This happens only on
+certain operating systems which have `gcc` compiled without `--with-as` and
+`--with-ld` flags. We need to make `as` visible manually in that case:
+
+```bzl
+# Create a symlink to system executable 'as'
+genrule(
+    name = "as",
```

Comment:
I've named this "toolchain_as". I'd recommend putting warning-free code in the example.